src/main/kotlin/no/nav/hm/grunndata/register/series/SeriesRegistrationController.kt

```diff
@@ -165,6 +165,26 @@ class SeriesRegistrationController(private val seriesRegistrationService: Series
         return HttpResponse.ok(updated)
     }
 
+    @Put("/series-to-inactive/{seriesUUID}")
+    suspend fun setPublishedSeriesToInactive(
+        @PathVariable seriesUUID: UUID,
+        authentication: Authentication,
+    ): HttpResponse<SeriesRegistrationDTO> {
+        val updated = seriesRegistrationService.setPublishedSeriesRegistrationStatus(seriesUUID, authentication, SeriesStatus.INACTIVE)
```

Comment:
Bør legge til validering i supplier-apiet på at at serien som oppdateres hører til innlogga suppliers, type som det er gjort her: 
`if (seriesRegistrationDTO.supplierId != authentication.supplierId()) {
            LOG.warn("SupplierId in request does not match authenticated supplierId")
            HttpResponse.unauthorized()
        }`
        
   Vi manglar dette på fleire eksisterande endepunkt også ser eg, eg opprettar oppgave på å fikse for dei og!